elliottlib/errata.py

```diff
@@ -32,6 +28,48 @@
 errata_xmlrpc = xmlrpc.client.ServerProxy(constants.errata_xmlrpc_url)
 
 
+class Advisory(Erratum):
+    """
+    Wrapper class of errata_tool.Erratum
+    """
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def ensure_state(self, target_state):
+        if target_state not in constants.errata_states:
+            raise ValueError(f'Desired state {target_state} is not a valid Errata state {constants.errata_states}')
+        if self.errata_state != target_state:
+            self.setState(target_state)
```

Comment:
i think you need to commit this before proceeding. it fails if you try it in a single pass, succeeds with this.
```suggestion
            self.setState(target_state)
            self.commit()
```